synapse/handlers/presence.py

```diff
@@ -523,6 +597,16 @@ def _start_polling_local(self, user, target_user):
         self._local_pushmap[target_localpart].add(user)
 
     def _start_polling_remote(self, user, domain, remoteusers):
+        """Subscribe a local user to presence updates for remote users on a
+        given domain.
```

Comment:
"from a given remote domain" - just to emphasise that the updates come from the remote, and it's definitely remote and not local.